crates/examples/infra/mod.rs

```diff
@@ -314,11 +311,7 @@ pub async fn run_orchestrator<
     OrchestratorArgs { url, config }: OrchestratorArgs<TYPES>,
 ) {
     println!("Starting orchestrator",);
-    let _result = hotshot_orchestrator::run_orchestrator::<
-        TYPES::SignatureKey,
-        TYPES::ElectionConfigType,
-    >(config, url)
-    .await;
+    let _result = hotshot_orchestrator::run_orchestrator::<TYPES::SignatureKey>(config, url).await;
```

Comment:
Curious, why do we bother storing this value?

Reply:
No idea lol, the change I made was to remove the generics. I can remove this though since it seems we're not using it